src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/LabelExtensions.cs

```diff
@@ -20,7 +20,12 @@ public static void UpdateTextColor(this TextView textView, ILabel label, Forms.C
 			else
 			{
 				textView.SetTextColor(textColor.ToNative());
-			}				
+			}
+		}
+
+		public static void UpdateGravity(this TextView textView, ILabel label)
```

Comment:
The gravity stuff for horizontal text alignment is left over from when we had to support down to API 15. API 17 and up has [setTextAlignment](https://developer.android.com/reference/android/view/View#setTextAlignment(int)). So the method for all of the Android Views should look like the stuff in [TextAlignmentExtensions](https://github.com/xamarin/Xamarin.Forms/blob/80a4dbfb1cd897f478c38d0aafa6a5d67940d2f3/Xamarin.Forms.Platform.Android/Extensions/TextAlignmentExtensions.cs). You should be able to take that, change the type from EditText to View, and drop all the < API 17 stuff.

Reply:
Fixed it up while rebasing.